src/components/OrgSettings/ActionItemCategories/OrgActionItemCategories.tsx

```
@@ -220,7 +223,11 @@ const OrgActionItemCategories: FC<InterfaceActionItemCategoryProps> = ({
       sortable: false,
       headerClassName: `${styles.tableHeader}`,
       renderCell: (params: GridCellParams) => {
-        return params.row.creator.firstName + ' ' + params.row.creator.lastName;
+        return (
+          <span data-testid="creatorName">
+            {params.row.creator.firstName + ' ' + params.row.creator.lastName}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)